src/libs/__tests__/pr.test.ts

```diff
@@ -161,7 +192,7 @@ describe('pr.ts', () => {
     await handlePullRequestMessage(options, projectName, 'test');
     expect(updateComment).toHaveBeenCalledWith({
       comment_id: 2,
-      body: '#### :tropical_drink: `preview` on myFirstProject/staging\n\n<details>\n<summary>Pulumi report</summary>\n\n<pre>\ntest\n</pre>\n\n</details>',
+      body: '#### :tropical_drink: `preview` on myFirstProject/staging\n\n<details>\n<summary>Pulumi report</summary>\n\n\n<pre>\ntest\n</pre>\n\n</details>',
```

Comment:
Why are we getting this extra newline everywhere?  I don't think we want that :thinking:

Reply:
https://github.com/pulumi/actions/pull/1252/files#r1725676206

I'll test this again and take a couple of screenshots so you can see exactly what I mean. I didn't notice any gratuitous spacing but I didn't look for it explicitly.

Reply:
I think it has something to do with coming right after the `</summary>` tag. The issue can be recreated in any comment.
<summary>This summary is good</summary>

⚠️ **Woohoo** look I'm bold

<summary>This is bad</summary>
⚠️ **Woohoo** look I'm bold?

Reply:
I think the extra `\n` needs to be in the tests